vault/datadog_checks/vault/vault.py

```diff
@@ -141,28 +143,34 @@ def get_config(self, instance):
 
         return config
 
-    def access_api(self, url, tags, params=None):
+    def access_api(self, url, path, tags, params=None):
         try:
-            response = self.http.get(url, params=params)
-            response.raise_for_status()
+            full_url = url + path
+            response = self.http.get(full_url, params=params)
             json_data = response.json()
+            response.raise_for_status()
         except requests.exceptions.HTTPError:
-            msg = 'The Vault endpoint `{}` returned {}.'.format(url, response.status_code)
-            self.service_check(self.SERVICE_CHECK_CONNECT, AgentCheck.CRITICAL, message=msg, tags=tags)
-            self.log.exception(msg)
-            raise ApiUnreachable
+            if path.endswith("/sys/health") and response.status_code in (429, 472, 473, 501, 503):
+                # Expected HTTP Error codes for /sys/health endpoint
+                # https://www.vaultproject.io/api/system/health.html
+                pass
```

Comment:
We should probably leave a debug line here.
@FlorianVeaux approve if there's going to be an RC

Reply:
added a debug line. would look like

```
2019-10-15 00:40:31 UTC | CORE | DEBUG | (pkg/collector/python/datadog_agent.go:120 in LogMessage) | vault:64fcda815f4f07c6 | (vault.py:167) | The Vault endpoint `http://localhost:8200/v1/sys/health` returned 503 - node is sealed.
```